play-services-nearby-core/src/main/kotlin/org/microg/gms/nearby/exposurenotification/AdvertiserService.kt

```diff
@@ -204,7 +212,11 @@ class AdvertiserService : LifecycleService() {
         advertising = false
         if (Build.VERSION.SDK_INT >= 26) {
             wantStartAdvertising = true
-            advertiser?.stopAdvertisingSet(setCallback as AdvertisingSetCallback)
+            try {
+                advertiser?.stopAdvertisingSet(setCallback as AdvertisingSetCallback)
+            } catch (e: SecurityException) {
+                Log.i(TAG, "Tried calling stopAdvertisingSet without android.permission.BLUETOOTH_ADVERTISE permission.", )
+            }
         } else {
             advertiser?.stopAdvertising(callback)
```

Comment:
I'm not sure since I don't really know that code but this case is the only one not try catched, is this right?

Reply:
I think there was a reason I didn't add this here but looking at it again I think it doesn't hurt to do the same as on the newer api here and just log a message :+1: .

Reply:
Added.